project/home/views.py

```diff
@@ -76,24 +78,85 @@ def next_month(d):
 
 
 def event(request, event_id=None):
-
+    instance = Event()
     if event_id:
         instance = get_object_or_404(Event, pk=event_id)
-        if not request.user.has_perm('view_event', instance):
-            messages.error(request, "You do not have permission to edit this event.")
-            return HttpResponse(status=204)
-            
-    else:
-        instance = Event()
-    
+
     form = EventForm(request.POST or None, instance=instance)
-    if request.POST and form.is_valid():
+    if request.method == 'POST' and form.is_valid():
```

Comment:
What was this change for?

Reply:
The change works the same thing. But, because when I merged sprint3 into my branch, and I kept the current changes.